packages/pancake-uikit/src/theme/index.ts

```diff
@@ -14,7 +17,10 @@ export interface PancakeTheme {
   alert: AlertTheme;
   colors: Colors;
   card: CardTheme;
-  nav: NavTheme;
+  nav: NavThemeType;
+  menuItem: MenuItemTheme;
+  subMenuItems: SubMenuItemsTheme;
+  dropdownMenuItem: DropdownMenuItemTheme;
```

Comment:
Are these usable by other components? If not maybe put them under the `nav` theme? Not sure though....

Reply:
There are not indeed, I thought that these theme files were specifics to components though.